src/github.com/getlantern/byteexec/byteexec_test.go

```diff
@@ -77,8 +82,9 @@ func testByteExec(t *testing.T, be *Exec) os.FileInfo {
 	if err != nil {
 		t.Errorf("Unable to run helloworld program: %s", err)
 	}
-	assert.Equal(t, "Hello world\n", string(out), "Should receive expected output from helloworld program")
+	assert.Equal(t, "Hello world"+linefeed, string(out), "Should receive expected output from helloworld program")
```

Comment:
On windows, we were getting a carriage return too, so I made this platform specific in order for test to pass.